cmd/zoekt-sourcegraph-indexserver/index_test.go

```diff
@@ -44,7 +44,7 @@ func TestGetIndexOptions(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	sg := newSourcegraphClient(u, "", 0)
+	sg := newSourcegraphClient(u, "", 0, 0)
```

Comment:
minor: this function likely deserves a struct describing its options. When I see two things of the same type next two each other I worry about mixups.